src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/ReducibleAnnotationBaseTest.java

```diff
@@ -85,7 +86,7 @@ public String getToolTestDataDir() {
 
     // NOTE: this code is mimicking the behavior of GATK3 combineGVCFS insofar as it is important for the annotations
```

Comment:
Is this comment still accurate, considering that you've changed "GATK3Concordance" to "AgainstExpected"?

Reply:
Good catch.